backend/app/routes/scraping.py

```
@@ -246,6 +247,44 @@ async def quick_scrape_infozagreb(
+
+
+@router.get("/visitopatija/quick", response_model=ScrapeResponse)
+async def quick_scrape_visitopatija(
+    max_pages: int = Query(1, ge=1, le=3, description="Number of pages to scrape (1-3 for quick scraping)")
+):
+    """Quick VisitOpatija.com scraping."""
+    try:
+        result = await scrape_visitopatija_events(max_pages=max_pages)
+        return ScrapeResponse(**result)
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"VisitOpatija scraping failed: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)